test/splitIncludeSourceMap.test.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/quotes */
+
+// Import 3rd party modules
+import test from 'ava'
+
+// Import module to be tested
+import { splitIncludeSourceMap } from '../src/index'
+
+test('should include original positions', t => {
+    const output = splitIncludeSourceMap([
+      "delimiter $$",
+      "SELECT * FROM table1$$",
+      "delimiter ;;",
+      "SELECT t.id FROM table2 t WHERE status = 'pending';;",
+      "DELIMITER ;",
+      "UPDATE table2 SET count=count+1 WHERE status = 'pending';"
+    ].join('\n'))
+    t.deepEqual(output, [
+      {
+        stmt: 'SELECT * FROM table1',
+        start: 12,
+        end: 35,
+      },
+      {
+        stmt: "SELECT t.id FROM table2 t WHERE status = 'pending'",
+        start: 48,
+        end: 101,
+      },
+      {
+        stmt: "UPDATE table2 SET count=count+1 WHERE status = 'pending'",
+        start: 113,
+        end: 171,
+      }
+    ])
```

Comment:
Please revise the test case to the following and fix the bug to pass the test.

I haven't really run the suggested code so bugs may exist, but I believe you know what the test trying to cover.
```suggestion
    const input = [
      "delimiter $$",
      "SELECT * FROM table1$$",
      "delimiter ;;",
      "SELECT t.id FROM table2 t WHERE status = 'pending';;",
      "DELIMITER ;",
      "UPDATE table2 SET count=count+1 WHERE status = 'pending';"
    ].join('\n')
    const output = splitIncludeSourceMap(input)
    t.is(output.length, 3)
    t.is(output[0].stmt, 'SELECT * FROM table1')
    t.is(output[1].stmt, "SELECT t.id FROM table2 t WHERE status = 'pending'")
    t.is(output[2].stmt, "UPDATE table2 SET count=count+1 WHERE status = 'pending'")
    for (const {stmt, start, end} of output) {
      t.is(stmt, sql.substring(start, end))
    }
```